### DISCUSSION TITLE
Is jde-robotics participating in gsoc this year(2023) ?

### DISCUSSION THREAD

**Discussion by GauravKumar9920** [JdeRobot/jderobot.github.io#234] [Q&A]
Is jde-robotics participating in gsoc this year(2023) ?

**Comment by jmplaza** [comment]
Yes, JdeRobot is participating. Take a look at the [JdeRobot GSoC-2023 webpage](https://jderobot.github.io/activities/gsoc/2023), in particular to the selection process and the [application instructions](https://jderobot.github.io/activities/gsoc/2023#application-instructions-for-gsoc-2023). 

The proposed projects, tentative mentors and required programming tests may still vary a little bit, but the general idea is clear.

**Reply by GauravKumar9920** [reply]
Hey, thanks !
I had a look at the assignments and they say 2022, is it gonna be updated or do we have to solve the same..

**Reply by jmplaza** [reply]
For those saying 2022, we will use them also for 2023. Watch out, there is a brand new REACT programming test for some projects. Cheers.

**Reply by GauravKumar9920** [reply]
Hey can we solve the ros2 challenge using ros2 humble ?

**Reply by jmplaza** [reply]
Sure. In fact, is the best release to do it

**Comment by jmplaza** [comment]
just wait a few days, we have to update the programming tests.

**Reply by omgupta113** [reply]
when will it be update?

**Reply by jmplaza** [reply]
Hi @omgupta113 , the programming tests have been just [updated](https://jderobot.github.io/activities/gsoc/2023#application-instructions-for-gsoc-2023), including a new REACT programming test for some projects. Cheers.

**Reply by staru09** [reply]
hello @jmplaza is there an irc or matrix channel for communication in JdeRobots or just this github discussion?

**Reply by jmplaza** [reply]
Hi @staru09 ,

we have just created a new GitHub Discussion for general JdeRobot questions and for GSoC too: [https://github.com/orgs/JdeRobot/discussions](https://github.com/orgs/JdeRobot/discussions). Feel free to ask there any question.

**Reply by jmplaza** [reply]
I have transferred there your message @staru09

**Reply by staru09** [reply]
ok cool
